### DISCUSSION TITLE
Journey to Quest NFT´s

### DISCUSSION THREAD

**Discussion by LorDDark6660** [stellar/sorobanathon#43] [General]
I would like to introduce you to my NFT series Journey to Quest. Illustration for the Live Quest story

![Journey began](https://user-images.githubusercontent.com/92062140/202287082-39cd328d-7681-4fe6-a286-1fad18947092.png)
Story Quest Prologue

https://litemint.com/items/GCAF2PLLYFDHVCCL7KI2ZP3BUACFDLTDLW6LLRNJ7JULRC7ANQ3LCRW7/SQLJourney0

![Find the key to open door](https://user-images.githubusercontent.com/92062140/202011463-3349932a-0a4c-43bb-b165-1306a6a0b085.png)
Story Quest 1

https://litemint.com/items/GDLZXALGOBG4NJROBLSLA6AAKK6POIHTTGTYRD4GAX3A57IRPQ2BLVHJ/SQLJourney1

![journey to live quest 3](https://user-images.githubusercontent.com/92062140/201917686-4be585cc-10d4-4444-a3a5-7273423de3b4.png)
Story Quest 2

https://litemint.com/items/GBV5VPWJSQWTV6KD5QM67Q4GUMQ4SGEAPVHW4O3NSX5L3KCB4MPD65JN/SQLJourney2

![Alchemist laboratory](https://user-images.githubusercontent.com/92062140/202712219-27c97e17-e6c4-4c71-b025-a8179b444257.png)
Story Quest 3

https://litemint.com/items/GDDXUUQ4JTA3D73YYK42APFMN6WB7IQMYW6OGDEQIQYJCKJ6YW6X3H5L/SQLJourney3

![ozias](https://user-images.githubusercontent.com/92062140/203145118-49604ddb-1bbf-4388-8342-7c510bbec574.png)
Story Quest 4

https://litemint.com/items/GCANDOTN3H5QCSEQXLPRCIGMKUKJ6CFSUI7IIF7MA4VILJU5P2N7PUNH/SQLJourney4

**Comment by kalepail** [comment]
Oh wow this is incredible!

**Reply by LorDDark6660** [reply]
Thank you, I'm still working on the first one...

**Comment by achallagundla** [comment]
WOWAH this si neat!! Looking forward to the rest of your series!

**Reply by LorDDark6660** [reply]
Thank you, I look forward to the next episode!

**Comment by stellarjohn** [comment]
BIG HYPE

**Reply by LorDDark6660** [reply]
Thank you, I just added NFT to the Prologue chapter...

**Reply by kalepail** [reply]
🔥

**Comment by briwylde08** [comment]
These are super cool and capture the vibe so well

**Reply by LorDDark6660** [reply]
Ty, I just added the alchemist laboratory

**Comment by stellarjohn** [comment]
Howdy LorDDark6660,

Congratulations!  Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM!  One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.
Please let us know if you have any questions.

Cheers!

**Reply by LorDDark6660** [reply]
Thank you, I would like to finish the whole collection, should I start a new discussion or add the last two NFTs when they are finished to this discussion

**Reply by stellarjohn** [reply]
Please start a new discussion.
